pyproject.toml

```
@@ -67,14 +67,15 @@ classifiers = [
     "Topic :: Utilities",
 ]
 dependencies = [
-    "nvidia-lm-eval==25.6.1",
+    "nvidia-lm-eval==25.7.1",
```

```suggestion
    "nvidia-lm-eval~=25.7.1",
```

Would this be safe or can nvidia-lm-eval also push breaking changes through patch releases?

---

The effective pin would be `>=25.7.1,<25.8.0`

---

@athitten you're probably referring to https://github.com/NVIDIA-NeMo/Eval/pull/63. Binding commons with nvidia-lm-eval==25.07.1 fixed the issue. So yes, this PR fixes breaking change.